### DISCUSSION TITLE
https://github.com/apache/incubator-superset not containes superset-frontend/src/visualizations/CountryMap/countries

### DISCUSSION THREAD

**Discussion by zsellami** [apache/superset#11690] [General]
Hello,

I want to add new countries in the country map tools.
The official documentation give a procedure to do that in https://superset.apache.org/docs/miscellaneous/country-map-tools

But I don't retrieve the folder **superset-frontend/src/visualizations/CountryMap/countries** and the file **superset-frontend/src/explore/controls.jsx** in the official github of the project.

How to do it ? I want to use the official github not a forke of the project

Regards
Zied Sellami

**Comment by rusackas** [comment]
Everything runs through that documented jupyter notebook now. You can add a country, make any touchups needed in the notebook , and the results of the notebook are everything you need to open a PR.
